### PR TITLE
Markdown update

### PR DESCRIPTION
**Pull request by LucianBuzzo**
- Don't memoize markdown-to-html function
- Allow inline html in markdown strings

### CODE REVIEW

File: __tests__/components/Markdown.spec.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {
+  Provider
+} from '../../dist'
+import {
+  Markdown
+} from '../../dist/extra/Markdown'
+
+describe('Markdown component', () => {
```
Comment (sradevski) [inline __tests__/components/Markdown.spec.js]: What was the reason to add markdown tests of our own? `marked` is much better tested and it feels redundant for us to also test the same things.

File: src/extra/Markdown.tsx [hunk 2]
```diff
@@ -13,10 +13,39 @@ const markedOptions = {
...
 	breaks: true,
 	headerIds: false,
-	sanitize: true,
+	// Input text is sanitized using `sanitize-html` prior to being transformed by
+	// `marked`
+	sanitize: false,
 };
 
-const render = memoize((text: string = '') => marked(text, markedOptions));
+const sanitizerOptions = {
```
Comment (sradevski) [inline src/extra/Markdown.tsx]: Maybe we can add a test or two to check if the input is properly sanitized.

File: __tests__/components/Markdown.spec.js (hunk 1 above)
Comment (LucianBuzzo) [reply __tests__/components/Markdown.spec.js]: @sradevski In short, its the same reasons we test anything.
Different combinations of sanitizers and marked options can lead to different results and rendering outputs. Additionally, by testing the output at the rendition level we can ensure that we maintain expected behaviour and a consistent API, even if we change the underlying implementation.

File: src/extra/Markdown.tsx (hunk 2 above)
Comment (LucianBuzzo) [reply src/extra/Markdown.tsx]: Good idea

**Pull request merged**